dali/python/nvidia/dali/external_source.py

```diff
@@ -229,15 +249,19 @@ class ExternalSource():
 Keyword Args
 ------------
 
-`cycle`: bool, optional
-    If set to True, the source will be wrapped.
+`cycle`: string or bool, optional
+    Specifies if and how to cycle through the source. It can be one of the following values:
+        * ``"no"`` or ``False`` - don't cycle; StopIteration is raised whe end of data is reached
+        * ``"quiet"`` or ``True`` - the data is repeated indefinitely
+        * ``"raise"`` - when the end of data is reached, StopIteration is raised, but the iteration is restarted on subsequent call
 
-    If set to False, StopIteration is raised when the end of data is reached.
     This flag requires that the ``source`` is a collection, for example, an iterable object where
     ``iter(source)`` returns a fresh iterator on each call or a gensource erator function.
     In the latter case, the generator function is called again when more data than was
     yielded by the function is requested.
 
+    Specifying ``"raise"`` can be used with DL framework iterators to create a notion of epoch.
```

Comment:
```suggestion
    Specifying ``"raise"`` can be used with DALI iterators to create a notion of epoch.
```
Those are called this way in our docs